libs/ngx-charts-on-fhir/tsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "./tsconfig.lib.json"
+  "extends": "../../tsconfig.base.json",
+  "include": ["**/*.ts"]
```

Comment:
does `tsconfig.lib.prod.json` need to extend `./tsconfig.lib.json` now?

Reply:
Yes `tsconfig.lib.prod.json` needs to extend `./tsconfig.lib.json` now and also `tsconfig.spec.json` needs to extend `tsconfig.json`.